Sources/UberCore/Authentication/Tokens/TokenManager.swift

```diff
@@ -22,171 +22,93 @@
-     
-     - returns: true if the accessToken was saved successfully, false otherwise
-     */
-    @discardableResult public static func save(accessToken: AccessToken, tokenIdentifier: String) -> Bool {
-        return self.save(accessToken: accessToken, tokenIdentifier: tokenIdentifier, accessGroup: Configuration.shared.defaultKeychainAccessGroup)
+    public init(keychainUtility: KeychainUtilityProtocol = KeychainUtility()) {
+        self.keychainUtility = keychainUtility
     }
     
-    /**
-     Saves the given AccessToken. 
-     Uses the default access token identifier & keychain access group defined by your
-     Configuration.
-     
-     Access Token is saved syncronously
-     
-     - parameter accessToken: The AccessToken to save
-     
-     - returns: true if the accessToken was saved successfully, false otherwise
-     */
-    @discardableResult public static func save(accessToken: AccessToken) -> Bool {
-        return self.save(accessToken: accessToken,  tokenIdentifier: Configuration.shared.defaultAccessTokenIdentifier, accessGroup: Configuration.shared.defaultKeychainAccessGroup)
+    // MARK: Save
+    
+    /// Saves the provided Access Token to the on device keychain using the supplied `identifier`
+    ///
+    /// - Parameters:
+    ///   - token: The Access Token to save
+    ///   - identifier: A string used to identify the Access Token upon retrieval
+    /// - Returns: A boolean indicating whether or not the save operation was successful
+    @discardableResult
+    public func saveToken(_ token: AccessToken, identifier: String = TokenManager.defaultAccessTokenIdentifier) -> Bool {
+        keychainUtility.save(token, for: identifier)
     }
     
-    //MARK: Delete
+    // MARK: Get
     
-    /**
-     Deletes the AccessToken for the givent tokenIdentifier and accessGroup. If no values
-     are supplied, it uses the defaults defined in your Configuration.
-
-     - parameter tokenIdentifier: The token identifier string to use (defaults to Configuration.shared.defaultAccessTokenIdentifier)
-     - parameter accessGroup:     The keychain access group to use (defaults to Configuration.shared.defaultKeychainAccessGroup)
-
-     - returns: true if the token was deleted, false otherwise
-     */
-    @discardableResult public static func deleteToken(identifier: String, accessGroup: String) -> Bool {
-        keychainWrapper.setAccessGroup(accessGroup)
-        deleteCookies()
-        let success = keychainWrapper.deleteObjectForKey(identifier)
-        if success {
-            NotificationCenter.default.post(name: Notification.Name(rawValue: tokenManagerDidDeleteTokenNotification), object: self)
-        }
-        return success
+    /// Retrieves an Access Token from the on device keychain
+    ///
+    /// - Parameter identifier: The identifier string used when saving the Access Token
+    /// - Returns: An optional Access Token if found
+    @discardableResult
```

Comment:
I think this should not be discardable right? The whole point of this function to get the token.